pinot-server/src/main/java/org/apache/pinot/server/starter/helix/BaseServerStarter.java

```diff
@@ -707,6 +711,23 @@ public void start()
     _serverQueriesDisabledTracker =
         new ServerQueriesDisabledTracker(_helixClusterName, _instanceId, _helixManager, serverMetrics);
     _serverQueriesDisabledTracker.start();
+
+    // Add metrics for consumer directory usage
+    serverMetrics.setOrUpdateGlobalGauge(ServerGauge.CONSUMER_DIR_USAGE, () -> {
+      List<File> serverConsumerDirs = instanceDataManager.getConsumerDirPaths();
+      final long[] totalSize = {0};
```

Comment:
why array? Single long is not enough?

Reply:
Changed to long. Previously used another lambda there that used the array!